models/public/quartznet/quartznet.md

```diff
@@ -0,0 +1,75 @@
+# quartznet
+
+## Use Case and High-Level Description
+
+QuartzNet model is a pair of encoder and decoder. QuartzNet’s design is based on the Jasper architecture,
```

Comment:
> QuartzNet model is a pair of encoder and decoder.

I think that's rather misleading, given that we merged them.

Furthermore, this is missing the most important information - what does this model actually _do_?

Reply:
Description updated